librz/crypto/p/crypto_aes.c

```diff
@@ -104,8 +160,8 @@ static bool aes_ecb_fini(RzCrypto *cry) {
 
 RzCryptoPlugin rz_crypto_plugin_aes = {
 	.name = "aes-ecb",
-	.author = "Karl Malbrain",
-	.license = "MS-PL",
+	.author = "Nettle project (algorithm implementation), pancake (plugin)",
```

Comment:
```suggestion
	.author = "Nettle project",
```

Reply:
I'd prefer to keep it this way for now... unless I rewrite even all the other functions.